guides/api-query-language-velo.md

```diff
@@ -0,0 +1,267 @@
+# API Query Language 
+
+The query language described in this article
+is implemented partially or in full by certain Wix Velo APIs that support query 
+capabilities.
+
+You may see some similarities between the Wix API Query Language
+and MongoQL. This is because the style of the Wix API Query Language
+is heavily influenced by MongoQL.
```

Comment:
The flow here I'm seeing is:

- paragraph 1: general
- paragraph 2: REST
- "Velo query techniqutes": Velo
- "API query language syntax": REST


I think it makes more sense to move this to the "API query language syntax" section and keep it with the rest of... REST 🙃

Reply:
@imnotashrimp Actually I was considering deleting the 2nd paragraph altogether because it is REST-y

API Query Language Syntax here is in the context of VELO --- it may be the REST syntax, but I think we can just use the term broadly here.  I wouldn't think of it as exclusively REST if our Velo queries are using it too.